app/enterprise/0.33-x/admin-gui/configuration/authentication.md

```diff
@@ -64,13 +64,16 @@ HTTP/1.1 201 Created
 
 Save this `"key": "62eb165c070a41d5c1b58d9d3d725ca1"` for use later. 
 
-Now that you have an Admin with an associated login Key, update the following in your Kong Configuration, then restart Kong:
+Now you need an RBAC user to associate to this admin. To add the super user see "[Bootstrapping the first RBAC user - the Super Admin](/enterprise/{{page.kong_version}}/rbac/examples/#bootstrapping-the-first-rbac-user-the-super-admin)".
```

Comment:
Could we stick with 80 chars per line please?

Reply:
Is there a comma missing at "To add a super user*,* see..."?

Reply:
Also, should we capitalize Super Admin?

Reply:
Make terminology consistent -- prefer "super admin" (not capitalized) over "super user".